app/src/main/java/org/wikipedia/pageimages/PageImage.kt

```diff
@@ -0,0 +1,45 @@
+package org.wikipedia.pageimages
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.wikipedia.dataclient.WikiSite
+import org.wikipedia.dataclient.mwapi.MwQueryPage
+import org.wikipedia.page.PageTitle
+
+@Parcelize
+data class PageImage(val title: PageTitle, val imageName: String) : Parcelable {
+
+    companion object {
+        @JvmField
+        val DATABASE_TABLE = PageImageDatabaseTable()
+
+        @JvmStatic
+        fun imageMapFromPages(wiki: WikiSite, titles: MutableList<PageTitle>, pages: MutableList<MwQueryPage>): Map<PageTitle?, PageImage> {
```

Comment:
Looks like the `Map<PageTitle?, PageImage>` can be definied as `Map<PageTitle, PageImage>` with an non-null `PageTitle`.